ethereum/p2p/src/main/java/org/hyperledger/besu/ethereum/p2p/rlpx/RlpxAgent.java

```diff
@@ -263,6 +272,8 @@ public CompletableFuture<PeerConnection> connect(final Peer peer) {
           }
         });
 
+    traceLambda(LOG, logConnectionsByIdToString());
```

Comment:
replace with 
`traceLambda(LOG, "{}", this::logConnectionsByIdToString)`
to avoid executing the costly method if log level is not trace.

The same applies to the other 2 `traceLambda` calls below